### DISCUSSION TITLE
I want more traffic on my website?

### DISCUSSION THREAD

**Discussion by RajaBOII333** [community/community#31827] [Other Feature Feedback, Questions, & Ideas]
I have a website jhaulage that provide containers to peoples. I want more traffic on it. Please tell me how can I get more traffic. Let me know how you treat newbies.  [redacted due to Code of Conduct].

**Comment by radar-blip** [comment]
You would have to do some [Search Engine Optimization](https://www.interaction-design.org/literature/topics/search-engine-optimization)
This will get your site indexed by google and other search engines and make for better visibility.
Research and experiment for best results.

Here is another link with much more detail: [SEO](https://developers.google.com/search/docs/beginner/seo-starter-guide?hl=en%2F&visit_id=637977843609597553-2863347569&rd=1)

**Comment by jfullstackdev** [comment]
same thing I asked in the past when I started putting projects on my profile and they were very hard to be discovered, I even tried Google Analytics to speed up the traffic

one thing is for sure, you need to be consistent, active and make sure your content is reliable

I started following others both in GitHub and Twitter, when I started doing these things, I received so many stars on my repos

**Comment by queenofcorgis** [comment]
Hi @RajaBOII333 , we’re glad you’re here 😀
Thanks for sharing your website with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
